aten/src/ATen/function_wrapper.py

```diff
@@ -135,27 +136,42 @@ def TypedDict(name, attrs, total=True):  # type: ignore
 
 # add non-virtual declaration to TensorBody.h
 TENSOR_METHOD_DECLARATION = CodeTemplate("""\
-${return_type} ${api_name}(${method_formals_with_defaults}) const;
+${return_type} ${api_name_prefix}${api_name}(${method_formals_with_defaults}) const;
 """)
+
+TENSOR_METHOD_DECLARATION_CONST = CodeTemplate("""\
+${return_type} ${api_name}(${type_method_formals}) const;
```

Comment:
This is weird af because `TENSOR_METHOD_DECLARATION` is already const. What's up with that?!